ChatCore/Models/MessagesRequest.swift

```diff
@@ -13,7 +13,7 @@ public struct MessagesRequest {
     public let count: Int
     public let includeInResult: Bool
     
-    public init(messageId: EntityIdentifier, direction: LoadingDirection, count: Int, includeInResult: Bool) {
+    public init(messageId: EntityIdentifier, direction: LoadingDirection, count: Int = 30, includeInResult: Bool = false) {
```

Comment:
You should use the existing constant from `Constants`

Reply:
:ajajajaj: